packages/auth/src/providers/cognito/apis/confirmResetPassword.ts

```
@@ -26,13 +26,13 @@ export async function confirmResetPassword(
 		!!code,
 		AuthValidationErrorCode.EmptyConfirmResetPasswordConfirmationCode
 	);
-	const config = Amplify.config;
+	const authConfig = AmplifyV6.getConfig().Auth;
```

Unrelated to this PR but just wanted to call it out. Seeing `getConfig()` in practice makes me think we need some error handling if the Library hasn't been configured yet 🤔 Maybe just throw an exception with that standard "Not configured" error & recovery suggestion.

Also maybe some memoization on `getConfig` to avoid the expensive JSON parsing (TODO)

---

I see, is ok if I add a TODO?

---

Okay with me!